js/js.go

```diff
@@ -48,7 +48,7 @@ func Class(class string) Value {
 	v := classes[class]
```

Comment:
```golang
key := class
if len(path) != 0 {
  key += "."+strings.Join(path, ".")
}
mu.RLock()
v := classes[key]
```

Reply:
Strangely I actually considered this last evening as I was mulling the problem over and yet didn't include it when I coded it up.  Not that the key is visible outside the js package but I put the class name after the path so it would look like the JSON dotted-notation if anyone ever saw it.

Reply:
It's actually the same here. The "class" argument is not a class name when the path is set. It's the first name in the chain. So it should be concatenated on the left side, not on the right. Here it doesn't matter that much, but it may confuse others looking at this code.

Reply:
If I'm instantiating an MDCChipSet in javascript I'd use:

```javascript
chipEl = document.CreateElement("div")
chipSet = mdc.chips.ChipSet.attachTo(chipEl)
```

So I thought it would make sense for the key to be "mdc.chips.ChipSet".  I don't care that much and will change it to be "ChipSet.mdc.chips" if you prefer that.

Reply:
No, you are totally right! Both of us are.

You mentioned the `mdc.chips.ChipSet` as the path. You won't write it as `Class("ChipSet", "mdc", "chips")`, right? The correct order is `Class("mdc", "chips", "ChipSet")`, so the first `class` argument is **not** a class name in case there are more than 1 argument to the function. So `class` should be concatenated to the left, since it's the `mdc` part.

Reply:
Right now calling ``js.Get("MDCChip", "mdc", "chips") works and in the TypeScript of the underlying material-components-web library, MDCChip is definitely the class.  The mdc and chips are in essence namespaces that are wrapped in closures.

Reply:
Should the signature just be ``js.Class(path ...string)``.  It simplifies the whole concatenation (if necessary) to ``key := strings.Join(path, ".")`` since if there's only one argument there will be no periods added.  Looking at the current signature I'm also noticing that there is no nil check on ``class``.

Reply:
Copied your key generation code from above into the PR.  Thanks!